express/parsers/apps/espresso/settings.py

```diff
@@ -7,6 +7,7 @@
 PHONON_DOS_FILE = "phonon_dos.out"
 PHONON_MODES_FILE = "normal_modes.out"
 AVERAGE_FILE = "avg.dat"
+HP_FILE = "__prefix__.Hubbard_parameters.dat"
```

Comment:
This output file name remains to be the same across versions (checked v6.7-7.2). We can leave it like this. `hp.x` calculation has not changed (at least in recent versions), however it depends on DFT+U SCF (`pw.x`) calculation, where we had major/breaking changes in v7.1.

Reply:
Since the base file name remains the same, I would leave it as it is. If the file name changes in future, we can add version suffix. Like in this case above:

<img width="367" alt="Screenshot 2023-12-07 at 7 46 27 PM" src="https://github.com/Exabyte-io/express/assets/31024886/03da9435-389c-4f0c-be57-2f94fb86c170">